src/Microsoft.ApplicationInsights.AspNetCore/Extensions/ApplicationInsightsExtensions.cs

```diff
@@ -132,21 +134,23 @@ public static IServiceCollection AddApplicationInsightsTelemetry(this IServiceCo
             {
                 services.TryAddSingleton<IHttpContextAccessor, HttpContextAccessor>();
 
-                services.AddSingleton<ITelemetryInitializer, ApplicationInsights.AspNetCore.TelemetryInitializers.DomainNameRoleInstanceTelemetryInitializer>();
-                services.AddSingleton<ITelemetryInitializer, AzureWebAppRoleEnvironmentTelemetryInitializer>();                
+                services
+                    .AddSingleton<ITelemetryInitializer, ApplicationInsights.AspNetCore.TelemetryInitializers.
+                        DomainNameRoleInstanceTelemetryInitializer>();
+                services.AddSingleton<ITelemetryInitializer, AzureWebAppRoleEnvironmentTelemetryInitializer>();
                 services.AddSingleton<ITelemetryInitializer, ComponentVersionTelemetryInitializer>();
                 services.AddSingleton<ITelemetryInitializer, ClientIpHeaderTelemetryInitializer>();
                 services.AddSingleton<ITelemetryInitializer, OperationNameTelemetryInitializer>();
-                services.AddSingleton<ITelemetryInitializer, ApplicationInsights.AspNetCore.TelemetryInitializers.OperationCorrelationTelemetryInitializer>();
                 services.AddSingleton<ITelemetryInitializer, SyntheticTelemetryInitializer>();
                 services.AddSingleton<ITelemetryInitializer, WebSessionTelemetryInitializer>();
                 services.AddSingleton<ITelemetryInitializer, WebUserTelemetryInitializer>();
                 services.AddSingleton<ITelemetryInitializer, AspNetCoreEnvironmentTelemetryInitializer>();
                 services.AddSingleton<ITelemetryInitializer, HttpDependenciesParsingTelemetryInitializer>();
                 services.TryAddSingleton<ITelemetryChannel, ServerTelemetryChannel>();
-                
+
                 services.AddSingleton<ITelemetryModule, DependencyTrackingTelemetryModule>();
-                services.ConfigureTelemetryModule<DependencyTrackingTelemetryModule>((module) => {                    
+                services.ConfigureTelemetryModule<DependencyTrackingTelemetryModule>((module, o) =>
```

Comment:
dont need 2nd argument o.

Reply:
I marked it as obsolete and want to get rid of it at some point. Do you want to keep it?